### PR TITLE
docs(payer-logo): update storybook for package: payer-logo, SHI-2910

### PR DESCRIPTION
**Pull request by cDavisAvaility**
@LauRoxx , does this seem right?

### CODE REVIEW

**Review by jordan-a-young — DISMISSED** [review]

File: packages/payer-logo/src/PayerLogo.js [hunk 1]
```diff
@@ -94,9 +94,12 @@ const PayerLogo = ({ spaceId, payerId, clientId, ...props }) => {
 };
 
 PayerLogo.propTypes = {
+  /** String. Required. Client ID to use to fetch the payer's logo. */
+  clientId: PropTypes.string.isRequired,
```
Comment (LauRoxx) [inline packages/payer-logo/src/PayerLogo.js:98]: the type and required/optional should be redundant within the description so I'd remove that.

File: packages/payer-logo/src/PayerLogo.stories.tsx [hunk 2]
```diff
@@ -9,6 +10,9 @@ export default {
   parameters: {
     docs: {
       // page: README,
+      description: {
+        component: "Easy to use component to display the payer's logo given the payer's ID",
```
Comment (LauRoxx) [inline packages/payer-logo/src/PayerLogo.stories.tsx]: just to really emphasize the state of the package, I'd either start or end the description with it's DEPRECATED status.

File: packages/payer-logo/src/PayerLogo.js (hunk 1 above)
Comment (cDavisAvaility) [reply packages/payer-logo/src/PayerLogo.js:98]: done

**Review by LauRoxx — DISMISSED** [review]

File: packages/payer-logo/src/PayerLogo.stories.tsx (hunk 2 above)
Comment (cDavisAvaility) [reply packages/payer-logo/src/PayerLogo.stories.tsx]: done

**Review by LauRoxx — APPROVED** [review]

**Review by jordan-a-young — APPROVED** [review]

**Pull request merged**